src/components/card_component/Card/Card.js

```diff
@@ -4,15 +4,15 @@ function tellColorName(house) {
   let color
 
   if (house === 'Slytherin') {
-    color = 'paragraph_house--green'
+    color = 'div_background--green'
```

Comment:
I don't really like the `div` in the class name 😄.

Reply:
Yeah, with my fuzzy mind I wasn't able to create a more convenient name :see_no_evil: I'm not quite sure, if I would use a `<div>` tag at all,  if I would spend more time to think about the positioning of all the elements. 
Would it be better to use `container__background--green` or 'content__background--green`?

Reply:
Just `bg-green` would be enough I guess. Then these can be so called utility classes.